lib/recurly/delivery.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+class Recurly_Delivery extends Recurly_Resource
+{
+  protected static $_writeableAttributes;
+
+  public static function init()
+  {
+    Recurly_Delivery::$_writeableAttributes = array(
```

Comment:
Let's skip this overhead and just return the array from `getWriteableAttributes()`